src/account/operations/output_consolidation.rs

```diff
@@ -1,13 +1,24 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use iota_client::block::output::{
-    unlock_condition::{AddressUnlockCondition, UnlockCondition},
-    BasicOutputBuilder, NativeTokensBuilder, Output,
+use iota_client::block::{
+    input::INPUT_COUNT_MAX,
+    output::{
+        unlock_condition::{AddressUnlockCondition, UnlockCondition},
+        BasicOutputBuilder, NativeTokensBuilder, Output,
+    },
 };
 #[cfg(feature = "ledger_nano")]
 use iota_client::secret::SecretManager;
 
+// Constants for the calculation of the amount of inputs we can use with a ledger nano
+#[cfg(feature = "ledger_nano")]
+const ESSENCE_SIZE_WITHOUT_IN_AND_OUTPUTS: usize = 49;
+#[cfg(feature = "ledger_nano")]
+const INPUT_SIZE_IN_ESSENCE: usize = 43;
+#[cfg(feature = "ledger_nano")]
+const MIN_OUTPUT_SIZE_IN_ESSENCE: usize = 46;
```

Comment:
Same question, I get 45 but I'm probably missing something

Reply:
I get 45 for a basic output, but then we need to also add the output type byte